src/main/java/teammates/common/util/HttpRequestHelper.java

```diff
@@ -82,21 +82,22 @@ public static String getValueFromRequestParameterMap(HttpServletRequest req, Str
 
     //TODO: rename to a better name
     public static String printRequestParameters(HttpServletRequest request) {
-        String requestParameters = "{";
+        StringBuilder requestParameters = new StringBuilder(); 
+        requestParameters.append('{');
         for (Enumeration<?> f = request.getParameterNames(); f.hasMoreElements();) {
             String paramet = new String(f.nextElement().toString());
-            requestParameters += paramet + "::";
+            requestParameters.append(paramet).append("::");
```

Comment:
Small request: I know you're not the one doing this, but let's rename `paramet` to something better, like `param`.